Java/HernanCano/README.md

```diff
@@ -0,0 +1,8 @@
+# **Hernan Cano**
+
++ Projects:
+    - [REST API](./API_REST/README.md)
```

Comment:
Decide if you are going to call it REST API or API REST, you shouldn't call the link REST API and the directory API_REST